proto/qdrant/cloud/hybrid/v1/hybrid_cloud.proto

```diff
@@ -365,6 +373,9 @@ enum HybridCloudEnvironmentConfigurationLogLevel {
 message HybridCloudEnvironmentStatus {
   // Timestamp when the hybrid cloud environment status was modified the last time.
   google.protobuf.Timestamp last_modified_at = 1 [(buf.validate.field).required = true];
+  // Set if the generate bootstrap commands has been called at least once.
+  // See HybridCloudService.GenerateBootstrapCommands for details.
+  bool bootstrap_commands_generated = 2;
   // Phase
   HybridCloudEnvironmentStatusPhase phase = 3 [(buf.validate.field).enum.defined_only = true];
   // Kubernetes version
```

Comment:
do we need to update this validation? When we generate the commands but we haven't received any real status yet, we won't have this information.